### PR TITLE
setup

### PR DESCRIPTION
**Pull request by Adrianzatreanu**

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Comment by sils** [comment]
would you mind writing proper commit messages?

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from distutils.core import setup
+setup(
+  name = 'coala-decorators',
+  packages = ['coala-decorators'],
+  version = '0.1',
+  description = 'All the coala decorators',
```
Comment (sils) [inline setup.py:6]: this package is no longer coala specific

Comment (Adrianzatreanu) [inline setup.py:6]: i have changed the commit message. what do you think would be a better description?

**Pull request merged**